docs/src/pages/components/icons/icons-ja.md

```diff
@@ -217,12 +217,15 @@ import SvgIcon from '@material-ui/core/SvgIcon';
 
 ```jsx
 import Icon from '@material-ui/core/Icon';
-import Typography from '@material-ui/core/Typography';
+import { visuallyHidden } from '@material-ui/system';
+import { makeStyles } from '@material-ui/core/styles';
+
+const classes = makeStyles({ visuallyHidden })();
 
 // ...
 
 <Icon>add_circle</Icon>
-<Typography variant="srOnly">Create a user</Typography>
+<span className={classes.visuallyHidden}>Create a user</span>
```

Comment:
We manage these translated files in https://translate.material-ui.com/. Changes done in GitHub will be lost. No need to bother :)